ClearBlade.js

```diff
@@ -456,8 +456,8 @@ n   * <p>{Number} [messagingPort] This is the default port used when connecting
       var usrid = getString(body);
       body = body.substring(usrid.length+2);
       var msgingHost = getString(body);
-      _this.setUser(email,tok);
-      _this.messagingURI = msgingHost;
+      _this.setUser(email,tok.substring(2, tok.length) + "==");
+      // _this.messagingURI = msgingHost;
```

Comment:
Is this being left in so that we can add it back later? If so, let's add a todo comment